examples/rails_example/Gemfile

```diff
@@ -0,0 +1,75 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "3.1.2"
+
+# The most relevant gem for this particular app
+gem 'featurehub-sdk'
```

Comment:
relevant line of code (I don't think anything else in the Gemfile is particularly relevant)